src/coreclr/minipal/Unix/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 set(SOURCES
     doublemapping.cpp
     dn-u16.cpp
+    ${CLR_SRC_NATIVE_DIR}/minipal/xoshiro128pp.c
```

Comment:
Should this be part of the minipal static library https://github.com/dotnet/runtime/blob/main/src/native/minipal/CMakeLists.txt instead this?

Reply:
yeah, probably. 👍

Reply:
Fixed